spec/rails_helper.rb

```diff
@@ -13,4 +14,10 @@
 
   config.infer_spec_type_from_file_location!
   config.filter_rails_from_backtrace!
+  Shoulda::Matchers.configure do |config|
```

Comment:
This setup block for Shoulda matcher should be set outside of the `RSpec.configure` block

Reply:
Tnx!